src/lang/ko_KR.rc

```diff
@@ -0,0 +1,1584 @@
+    DEFPUSHBUTTON "확인", IDOK, 70, 55, 60, 14
+}
+
+IDD_EGA DIALOG 0, 0, 400, 225
+CAPTION "프로그래밍 언어 EGA"
+STYLE WS_POPUPWINDOW | WS_CAPTION | WS_THICKFRAME
+FONT 9, "MS Shell Dlg"
+{
+    GROUPBOX "EGA 콘솔", grp1, 5, 5, 390, 195
+    EDITTEXT edt1, 10, 20, 380, 170, ES_READONLY | ES_AUTOVSCROLL | ES_MULTILINE | WS_VSCROLL | NOT WS_TABSTOP
+    RTEXT "EGA>", stc1, 10, 208, 20, 10
+    COMBOBOX cmb1, 35, 205, 300, 300, CBS_NOINTEGRALHEIGHT | CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    DEFPUSHBUTTON "엔터(&E)", IDOK, 345, 205, 50, 15
+}
+
+IDD_REPLACEFONTS DIALOG 0, 0, 185, 180
+CAPTION "대화 상자 글꼴 바꾸기"
+STYLE DS_CENTER | DS_MODALFRAME | WS_POPUPWINDOW | WS_CAPTION
+FONT 9, "MS Shell Dlg"
+{
+    LTEXT "글꼴 #&1:", -1, 5, 7, 44, 12
+    COMBOBOX cmb1, 55, 5, 125, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    ICON IDI_ARROW, -1, 30, 25, 0, 0
+    COMBOBOX cmb2, 55, 30, 125, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    LTEXT "글꼴 #&2:", -1, 5, 57, 44, 12
+    COMBOBOX cmb3, 55, 55, 125, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    ICON IDI_ARROW, -1, 30, 75, 0, 0
+    COMBOBOX cmb4, 55, 80, 125, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    LTEXT "글꼴 #&3:", -1, 5, 105, 44, 12
+    COMBOBOX cmb5, 55, 105, 125, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    ICON IDI_ARROW, -1, 30, 125, 0, 0
+    COMBOBOX cmb6, 55, 130, 125, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    DEFPUSHBUTTON "확인", IDOK, 75, 160, 50, 14
+    PUSHBUTTON "취소", IDCANCEL, 130, 160, 50, 14
+    PUSHBUTTON "초기화(&R)", psh1, 5, 160, 55, 14
+}
+
+IDD_DROPDOWNPOPUP DIALOG 0, 0, 120, 150
+CAPTION " "
+STYLE WS_POPUP | WS_DLGFRAME | NOT WS_BORDER
+FONT 9, "MS Shell Dlg"
+{
+    LISTBOX lst1, 0, 0, 120, 150, LBS_WANTKEYBOARDINPUT | LBS_NOINTEGRALHEIGHT | LBS_HASSTRINGS | WS_VSCROLL | WS_TABSTOP, WS_EX_CLIENTEDGE
+}
+
+IDD_DFMSETTINGS DIALOG 0, 0, 210, 135
+CAPTION "Delphi DFM 설정"
+STYLE DS_CENTER | DS_MODALFRAME | WS_POPUPWINDOW | WS_CAPTION
+FONT 9, "MS Shell Dlg"
+{
+    LTEXT "다음 설정은 Delphi DFM과 관련이 있습니다", -1, 35, 5, 171, 20
+    ICON IDI_DFMICON, -1, 5, 5, 0, 0
+    RTEXT "코드페이지(&C):", -1, 6, 32, 70, 12
+    COMBOBOX cmb1, 85, 30, 120, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    AUTOCHECKBOX "원시 문자열 주석 삽입(&I)", chx1, 5, 50, 200, 14
+    AUTOCHECKBOX "유니 코드와 UTF-8을 사용하지 마십시오(&U)", chx2, 5, 70, 200, 14
+    DEFPUSHBUTTON "확인", IDOK, 80, 115, 60, 14
+    PUSHBUTTON "취소", IDCANCEL, 145, 115, 60, 14
+}
+
+//////////////////////////////////////////////////////////////////////////////
+// RT_VERSION
+
+1 VERSIONINFO
+FILEVERSION     5, 4, 9, 0
+PRODUCTVERSION  5, 4, 9, 0
+FILEOS          0x40004
+FILETYPE        0x1
+{
+    BLOCK "StringFileInfo"
+    {
+        BLOCK "040904E4"
```

Comment:
```suggestion
        BLOCK "041203B5"
```
Can you verify this value for Korean? Thanks.
https://docs.microsoft.com/en-us/windows/win32/menurc/versioninfo-resource